dashboards/vmagent.json

```diff
@@ -3691,6 +3691,108 @@
           ],
           "title": "Restarts ($job)",
           "type": "timeseries"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "$ds"
+          },
+          "description": "Shows how many rows were ignored on insertion due to corrupted or out of retention timestamps.",
```

Comment:
```suggestion
          "description": "Shows how many samples were ignored due to various reasons. See the reason of rejection application logs.",
```

Reply:
"corrupted or out of retention timestamps" this part needs to updated in cluster version

the whole comment needs to be updated in single-node version, as it could scrape and receive writes as well

Reply:
done